controllers/cluster_controller.go

```diff
@@ -65,6 +65,8 @@ const (
 // ClusterReconciler reconciles a Cluster object.
 type ClusterReconciler struct {
 	Client           client.Client
+	
+	//Label value to filter events for reconciliation
```

Comment:
Maybe 
```suggestion
	// Label value that the controller watches to reconcile cluster-api objects. If unspecified, the controller watches for all cluster-api objects.
```

This keeps the comment directly the same as on the CLI flag in main.go https://github.com/kubernetes-sigs/cluster-api/blob/2b81f62ad668ddb010cb37d734ca0677c752a13c/main.go#L118